### ISSUE TITLE
incoming / outgoing traffic measurement 

### ISSUE THREAD

**Issue by WoLfulus** [nodejitsu/node-http-proxy#525]
Hello, sorry if this isn't the right place to ask this, but, how can I actually measure how much bytes are being sent/received for a specific domain? 

For example: 

I've set 3 servers on the route table.

website1.com -> localhost:3001
website2.com -> localhost:3002
website3.com -> localhost:3003

I want to calculate how much bandwidth is taken for each host per-request (I'll sum them later)

Request 1 to website1.com:
-> Request = 384 bytes (header + body)
-> Response = 658 bytes (header + body)

Request 2 to website1.com
-> Request = 460 bytes (header + body)
-> Response = 1560 bytes (header + body)

Basically I need to calculate how much bandwidth is being takes for each host, and since I'm using node-http-proxy to actually serve them from the same server/port (80), I find it easier to calculate this inside the proxy application. How can I accomplish this?
